packer_images/probate_logstash.conf.in

```diff
@@ -1,19 +1,181 @@
+            "date_of_death_classification" => "PUBLIC"
+            "date_of_birth_classification" => "PUBLIC"
+            "alias_names_classification" => "PUBLIC"
+            "record_id_classification" => "PUBLIC"
+            "imported_to_ccd_classification" => "PUBLIC"
+            "legacy_case_type_classification" => "PUBLIC"
+            "ccd_case_no_classification" => "PUBLIC"
+            "legacyId_classification" => "PUBLIC"
+            "security_classification" => "PUBLIC"
+            "state" => "ReadOnly"
+            "case_type_id" => "Legacy"
+            "jurisdiction" => "PROBATE"
+        }
+        remove_field => ["src"]
+    }
+    mutate {
+        rename => { "probate_number" => "[data][probate_number]" }
+        rename => { "deceased_forenames" => "[data][deceasedForenames]" }
+        rename => { "deceased_surname" => "[data][deceasedSurname]" }
+        rename => { "date_of_death" => "[data][deceasedDateOfDeath]" }
+        rename => { "date_of_birth" => "[data][deceasedDateOfBirth]" }
+        rename => { "alias_names" => "[data][alias_names]" }
+        rename => { "record_id" => "[data][record_id]" }
+        rename => { "imported_to_ccd" => "[data][imported_to_ccd]" }
+        rename => { "legacy_case_type" => "[data][legacy_case_type]" }
+        rename => { "ccd_case_no" => "[data][ccdCaseId]" }
+        rename => { "legacyId" => "[data][legacyId]" }
 
+        rename => { "probate_number_classification" => "[data_classification][probate_number]"}
+        rename => { "deceased_forenames_classification" => "[data_classification][deceasedForenames]"}
+        rename => { "deceased_surname_classification" => "[data_classification][deceasedSurname]"}
+        rename => { "date_of_death_classification" => "[data_classification][deceasedDateOfDeath]"}
+        rename => { "date_of_birth_classification" => "[data_classification][deceasedDateOfBirth]"}
+        rename => { "alias_names_classification" => "[data_classification][alias_names]"}
+        rename => { "record_id_classification" => "[data_classification][record_id]"}
+        rename => { "imported_to_ccd_classification" => "[data_classification][imported_to_ccd]"}
+        rename => { "legacy_case_type_classification" => "[data_classification][legacy_case_type]" }
+        rename => { "ccd_case_no_classification" => "[data_classification][ccdCaseId]" }
+        rename => { "legacyId_classification" => "[data_classification][legacyId]" }
+    }
+    # mutate { remove_field => [ "@timestamp", "@version" ] }
+    mutate {
+        add_field => { "[data][legacyId]" => "%{id}"}
+    }
+    mutate {
+        add_field => { "index_id" => "%{case_type_id}_cases" }
+    }
+    mutate {
+        lowercase => [ "index_id" ]
+    }
+}
 output {
     elasticsearch {
-			hosts => ["${ES_DATA_NODES_URL}"]
-			sniffing => false
-			index => "%{[index_id]}"
-			document_type => "_doc"
-			document_id => "%{id}"
+        hosts => [ES_DATA_NODES_URL]
```

Comment:
hosts => [ES_DATA_NODES_URL] might not work. Don't remember exactly now but to make this work I had to use ["${ES_DATA_NODES_URL}"]